eth_account/hdaccount/_utils.py

```diff
@@ -0,0 +1,27 @@
+import hashlib
```

Comment:
There's an established precedent for doing it this way, and since we plan to yank this out in favor of py-hdwallet in the near term future, I think it makes the most sense to leave it here since it will make that extraction easier.